lib/PuppeteerSharp/Frame.cs

```diff
@@ -257,7 +257,14 @@ public async Task<JSHandle> EvaluateFunctionHandleAsync(string function, params
         /// Gets the <see cref="ExecutionContext"/> associated with the frame.
         /// </summary>
         /// <returns><see cref="ExecutionContext"/> associated with the frame.</returns>
-        public Task<ExecutionContext> GetExecutionContextAsync() => _contextResolveTaskWrapper.Task;
+        public Task<ExecutionContext> GetExecutionContextAsync()
+        {
+            if (Detached)
```

Comment:
@kblok this is the only read change in the file
I think the rest are from a Ctrl+K+D in vs...